distribution/std-lib/Base/src/Data/Vector.enso

```diff
@@ -226,6 +332,71 @@ type Vector
     to_json : Json.Array
     to_json = Json.Array (this.map to_json)
 
+    ## Get the first element from the vector.
+
+       > Example
+         The following code returns 1.
+             [1, 2, 3, 4].head
+       > Example
+         Empty vectors return `Nothing`.
+             [].head == Nothing
+    head : Any | Nothing
+    head = if this.length >= 1 then this.at 0 else Nothing
+
+    ## Get all elements in the vector except the first.
+
+       > Example
+         The following code returns [2, 3, 4].
+             [1, 2, 3, 4].tail
+       > Example
+         Empty vectors return `Nothing`.
+             [].tail == Nothing
+    tail : Vector | Nothing
+    tail = if this.length >= 1 then this.drop 1 else Nothing
+
+    ## Get the all elements in the vector except the last.
+
+       > Example
+         The following code returns [1, 2, 3].
+             [1, 2, 3, 4].init
+       > Example
+         Empty vectors return `Nothing`.
+             [].init == Nothing
+    init : Vector | Nothing
+    init = if this.length >= 1 then this.drop_right 1 else Nothing
```

Comment:
I think `init` and `tail` should be consistent with `take` and `drop`. 

```
[].drop 1 == []
```
but
```
[].tail == Nothing
```

Both should return either empty Vector or Nothing.